nginx_proxy/config.json

```diff
@@ -34,7 +35,8 @@
       "active": "bool",
       "default": "str",
       "servers": "str"
-    }
+    },
+    "security_mode": "str"
```

Comment:
I'd suggest doing some basic schema validation here since you're only using three options (On, Off, DetectionOnly), otherwise you may end up with some unparsable configuration in the modsecurity.conf file later.

"security_mode" : "match(On|Off|DetectionOnly)"
or
"security_mode" : "list(On|Off|DetectionOnly)"

I don't know much about Modsec's config case-sensitivity, so you may need to do "match" with a regex to handle case-insensitivity if you want to allow that.

Reply:
This should already be updated to `"security_mode": "list(DetectionOnly|On|Off)"`. In an older commit it was `"security_mode": "str"`, so maybe an older revision got selected during the review.

Reply:
I see it now. Yeah, I think I might have been in an older commit by mistake. Disregard.